scss/prototype/_opacity.scss

```diff
@@ -0,0 +1,19 @@
+// Foundation for Sites by ZURB
+// foundation.zurb.com
+// Licensed under MIT Open Source
+
+////
+/// @group prototype-opacity
+////
+
+/// Opacity Mixin: Easily create opacity to an element
+/// @param {Number} $opacity[] Opacity/Transparency of an element
+@mixin opacity($opacity) {
+  @if ($opacity >= 0 and $opacity <= 1) {
+    opacity: $opacity;
+    $opacity-ie: $opacity * 100;
+    filter: alpha(opacity=$opacity-ie); //IE8
+  } @else {
+    @warn 'Opacity should be between 0.0 and 1.0';
```

Comment:
@IamManchanda you could say that about every single CSS property though (eg `float: top;`) 😄